api/src/main/java/io/opentelemetry/trace/DefaultSpan.java

```diff
@@ -91,6 +91,26 @@ public void setAttribute(String key, boolean value) {
     Utils.checkNotNull(key, "key");
   }
 
+  @Override
+  public void setAttribute(String key, String... values) {
```

Comment:
Please add tests in `DefaultSpanTest.doNotCrash` for passing `(String[]) null` and so on.

Reply:
Also, what should happen if `null` or empty strings are passed inside the array? I'm strongly against dropping them silently as done with #771 for single strings, as this would preclude use cases such as having corresponding key and value arrays where `value_array[i]` is the value for `key_array[i]`.